linera-core/src/updater.rs

```diff
@@ -162,83 +162,74 @@ where
     S: Storage + Clone + Send + Sync + 'static,
     ViewError: From<S::ContextError>,
 {
-    async fn send_certificate(
+    async fn send_optimized_certificate(
```

Comment:
So "optimized" means we don't retry if the validator is missing bytecode? Would be good to say that in a method comment.

Reply:
I meant to say that we try the "lite" one first

Reply:
maybe "send_certificate_optimistically" is better?

Reply:
Ah, makes sense! I'm fine with either name; I just misunderstood at first.